packages/react-components/react-popover/src/components/Popover/usePopover.ts

```diff
@@ -120,14 +120,14 @@ export const usePopover_unstable = (props: PopoverProps): PopoverState => {
   const { findFirstFocusable } = useFocusFinders();
 
   React.useEffect(() => {
-    if (open && positioningRefs.contentRef.current) {
+    if (open && positioningRefs.contentRef.current && !props.unstable_disableAutoFocus) {
```

Comment:
```suggestion
    if (props.unstable_disableAutoFocus) {
      return;
    }

    if (open && positioningRefs.contentRef.current) {
```

nit: can you please do an early return instead?